### ISSUE TITLE
Not compatible with Django==1.8

### ISSUE THREAD

**Issue by blfpd** [makinacorpus/django-appypod#1]
I've got this traceback when lauching runserver:

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/debug_toolbar/apps.py", line 15, in ready
    dt_settings.patch_all()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/debug_toolbar/settings.py", line 232, in patch_all
    patch_root_urlconf()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/debug_toolbar/settings.py", line 220, in patch_root_urlconf
    reverse('djdt:render_panel')
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 550, in reverse
    app_list = resolver.app_dict[ns]
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 352, in app_dict
    self._populate()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 308, in _populate
    for name in pattern.reverse_dict:
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 338, in reverse_dict
    self._populate()
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 326, in _populate
    lookups.appendlist(pattern.callback, (bits, p_pattern, pattern.default_args))
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 247, in callback
    self._callback = get_callable(self._callback_str)
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 106, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/batisteo/ps/book/views.py", line 3, in <module>
    from djappypod.response import OdtTemplateResponse
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/djappypod/response.py", line 3, in <module>
    from .odt import get_template
  File "/home/batisteo/.virtualenvs/ps/local/lib/python2.7/site-packages/djappypod/odt.py", line 9, in <module>
    from django.template.loader import find_template, LoaderOrigin
ImportError: cannot import name find_template
```

The [documentation for the 1.8](https://docs.djangoproject.com/en/1.8/releases/1.8/) says:

> If you’ve been relying on private APIs exposed in the django.template module, you may have to import them from django.template.base instead.
> Also private APIs django.template.base.compile_string(), **django.template.loader.find_template()**, and django.template.loader.get_template_from_string() were removed.

**Comment by leplatrem** [comment]
Hello Baptiste, nice to hear some feedback from you :)
I didn't know you were using django-appypod !

It looks like you identified the problem...
I'm not sure @gutard has plans to upgrade Geotrek to Django 1.8 in a near future.. So maybe you could open a pull-request to add a small safety check around the imports ?
We'll be happy to review it !

**Comment by leplatrem** [comment]
...oh. I forgot the most important : **Thank you for taking the time to report this!**

**Comment by blfpd** [comment]
Hello Mathieu! You're welcome!
I'm new to templates customisation, and Django 1.8 introduced the Engine class to be able to use Jinja2, or so.

I'm half way of the fix, to put in the except block:

``` python
from django.template import engines
find_template = engines['django'].engine.find_template
```

…but what I upload is just an empty file (13b).

I may stick to Dj1.7 till sombody (@gutard?) get motivated to have a look.

Btw, any alternative to produce an ODT out of Django?

Thanks

**Comment by leplatrem** [comment]
> I'm half way of the fix, to put in the except block:

Looks like you got it ! Open a pull-request, we'll guide you !

> …but what I upload is just an empty file (13b).

What do you mean ? (_on peut se parler en français, si tu préfères_)

> Btw, any alternative to produce an ODT out of Django?

Nope :)

**Comment by keithcallenberg** [comment]
I think @batisteo means that when he implements this correction, the odt that is returned is only 13 bytes. I am similarly getting a file that is just 12 bytes with Django 1.8.4 when I implement this.

**Comment by gutard** [comment]
Fixed in version 1.0.0
